src/main/kotlin/org/rust/ide/actions/RsShowControlFlowGraphAction.kt

```diff
@@ -0,0 +1,77 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.actions
+
+import com.intellij.openapi.actionSystem.AnAction
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.actionSystem.CommonDataKeys
+import com.intellij.openapi.actionSystem.DataContext
+import com.intellij.openapi.application.PathManager
+import com.intellij.openapi.ui.Messages
+import com.intellij.util.io.createDirectories
+import com.intellij.util.io.delete
+import org.rust.lang.core.cfg.ControlFlowGraph
+import org.rust.lang.core.psi.RsBlock
+import org.rust.lang.core.psi.RsFunction
+import org.rust.lang.core.psi.ext.ancestorOrSelf
+import java.awt.Desktop
+import java.io.File
+import java.io.IOException
+import java.nio.file.Paths
+
+class RsShowControlFlowGraphAction : AnAction() {
```

Comment:
I think this action should be available only for Rust files.
You can override `update` method for this purpose:
```kotlin
override fun update(event: AnActionEvent) {
    val file = event.getData(CommonDataKeys.PSI_FILE)
    event.presentation.isEnabled = file is RsFile
}
```